src/librbd/internal.cc

```diff
@@ -2273,11 +2279,16 @@ namespace librbd {
   void close_image(ImageCtx *ictx)
   {
     ldout(ictx->cct, 20) << "close_image " << ictx << dendl;
+    if (ictx->copyup_finisher != NULL) {
```

Comment:
should this be after the readahead cleanup, so that any copyups triggered by readahead can still complete?